carbonmark-data/lib/constants.ts

```diff
@@ -37,7 +37,9 @@ export const urls = {
     dailyAggregatedCredits: `${api_url}/credits/agg/daily`,
     aggregatedCredits: `${api_url}/credits/agg`,
     prices: `${api_url}/prices`,
+    tokens: `${api_url}/tokens`,
   },
 };
 
 export const BRIDGES: Array<Bridge> = ["toucan", "c3", "moss"];
+export const TOKENS: Array<Token> = ["bct", "nct", "mco2", "ubo", "nbo"];
```

Comment:
We can use this approach to only define these once: https://stackoverflow.com/a/64174790/1868281

Reply:
Good stuff. Done in f08c1f69